Epic7Math/Program.cs

```diff
@@ -56,11 +44,18 @@ static void Main(string[] args)
                 //}
                 totalboost++;
             }
-            Console.WriteLine(totalboost);
             Debug.WriteLine(totalboost);
 
         }
 
+        private static void checkInput(string[] args)
+        {
+            if (args.Length < 6) {
```

Comment:
Not sure why 6. But if less `totalboost` will be 0.

Reply:
I did not wanted to make it as args because I'm running this in parallel with another project that with OCR will be able to read the character list. In theory when someone put's this as a service in a website, main should become a method that they can call with the list. It's a good idea though :-)

Reply:
This way its easier to use it from CLI. Anyway you can create CLI arg `--list` and if its present load the list from CLI otherway use hardcoded one.

Reply:
But why 6?